src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -47,21 +47,23 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         private readonly IHost extensionHost;
         private readonly IActiveSolutionTracker solutionTracker;
         private readonly IConfigurationProvider configurationProvider;
-        private readonly IVsMonitorSelection vsMonitorSelection;
         private readonly IBoundSolutionGitMonitor gitEventsMonitor;
+        private readonly IVsUIServiceOperation vSServiceOperation;
         private readonly ILogger logger;
-        private readonly uint boundSolutionContextCookie;
 
         public event EventHandler<ActiveSolutionBindingEventArgs> PreSolutionBindingChanged;
+
```

Comment:
My auto format removed the white spaces but I'll leave it considering we usually format it like this